certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/PdfConverter.java

```diff
@@ -13,6 +19,12 @@
     public static void convertor(File htmlSource, String certUuid, String directory) {
         File file = new File(directory, certUuid + ".pdf");
         try {
+            boolean licenseEnabled = Boolean.parseBoolean(System.getenv(JsonKey.ITEXT_LICENSE_ENABLED));
+            if(licenseEnabled) {
```

Comment:
insdie if we can directly pass : Boolean.parseBoolean(System.getenv(JsonKey.ITEXT_LICENSE_ENABLED))
* i think we need to handle both scenario (if license is enabled and path is provided then only load it ), in other case use default  and add one logger that will let you know what user has provided.

Reply:
@manzarul even though if license is loaded or not remaining works same. added logger.
Done accordingly.